packages/perseus/src/widgets/interactive-graphs/utils.test.ts

```diff
@@ -119,4 +122,23 @@ describe("replaceOutsideTeX", () => {
             "\\text{Square }A\\text{ is }B\\text{ also}",
         );
     });
+
+    test("with a real $ inside a regular string", () => {
+        const string = "This sandwich is \\$12";
+        const convertedString = replaceOutsideTeX(string);
+
+        // For some reason, the parser starts a new text block once it hits
+        // the '$' character. The result is still the same though.
+        // This is functionally equal to "\\text{This sandwich is $12}"
```

Comment:
I don't think screenreaders will read the text in exactly the same way if it's split up. [Specific example on the TeX playground](https://www.khanacademy.org/devadmin/tex#JTVDdGV4dCU3QlRoaXMlMjBzYW5kd2ljaCUyMGlzJTIwJTdEJTVDdGV4dCU3QiU1QyUyNCU3RCU1Q3RleHQlN0IxMiU3RCUyMCU1QyU1QyUwQSU1Q3RleHQlN0JUaGlzJTIwc2FuZHdpY2glMjBpcyUyMCU1QyUyNDEyJTdE)

VoiceOver on Edge reads "This sandwich is ispace dollar twelve" for the first line, and "This sandwich is dollar twelve" for the second line.